### PR TITLE
DFR-1503: FR Send order event - CCD address field for applicantAddres…

### PR DESCRIPTION
**Pull request by srochani**
…s validation

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateConsentedCaseController.java [hunk 1]
```diff
@@ -220,7 +220,8 @@ private void removePropertyAdjustmentDetails(Map<String, Object> caseData) {
 
 
     private void updateApplicantOrSolicitorContactDetails(Map<String, Object> caseData) {
-        if (equalsTo((String) caseData.get(APPLICANT_REPRESENTED), "No")) {
```
Comment (rishikrsharma) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateConsentedCaseController.java:223]: better to use Objects.toString method and set the default there itself. no cast needed then

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/SendOrderContestedAboutToSubmitHandler.java [hunk 2]
```diff
@@ -59,9 +61,13 @@ public AboutToStartOrSubmitCallbackResponse handle(CallbackRequest callbackReque
...
 
-        printAndMailGeneralOrderToParties(caseDetails, userAuthorisation);
-        printAndMailHearingDocuments(caseDetails, userAuthorisation);
-        stampFinalOrder(caseDetails, userAuthorisation);
+        try {
+            printAndMailGeneralOrderToParties(caseDetails, userAuthorisation);
+            printAndMailHearingDocuments(caseDetails, userAuthorisation);
+            stampFinalOrder(caseDetails, userAuthorisation);
+        } catch (InvalidCaseDataException e) {
+            return AboutToStartOrSubmitCallbackResponse.builder().errors(ImmutableList.of(e.getMessage())).build();
```
Comment (rishikrsharma) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/SendOrderContestedAboutToSubmitHandler.java:69]: do we need to caseData also in reponse?

Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/SendOrderContestedAboutToSubmitHandler.java:69]: No, if error then we don't require casedata here

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateConsentedCaseController.java (hunk 1 above)
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateConsentedCaseController.java:223]: If caseData.get(APPLICANT_REPRESENTED) return null then toString method throw nullPointerException

**Review by rishikrsharma — APPROVED** [review]

**Pull request merged**